src/index.css

```diff
@@ -28,7 +28,7 @@ body {
     place-items: center;
     min-width: 320px;
     min-height: 100vh;
```

Comment:
Setting min-width and min-height on the body is unnecessary because the body already occupies the full available width and height of the viewport by default.
Using these properties on the body can potentially cause layout issues on smaller screens (Like mine) or in responsive designs.

I removed the lines locally and it got rid of the horizontal scrolling.